app/src/main/kotlin/org/vinaygopinath/launchchat/screens/main/MainActivity.kt

```diff
@@ -99,6 +111,33 @@ class MainActivity : AppCompatActivity() {
         }
     }
 
+private fun showPhoneNumberSelectionDialog(phoneNumbers: List<String>, onNumberSelected: (String) -> Unit) {
+    val items = phoneNumbers.toTypedArray()
+    val builder = AlertDialog.Builder(this)
+    builder.setTitle(R.string.dialog_title_multiple_phone_numbers)
+
+    val inflater = this.layoutInflater
+    val dialogView = inflater.inflate(R.layout.custom_dialog, null)
+    builder.setView(dialogView)
+
+    val listView = dialogView.findViewById<ListView>(R.id.listView)
+    val textView = dialogView.findViewById<TextView>(R.id.textView)
+    textView.text = getString(R.string.dialog_message_multiple_phone_numbers)
+
+    val adapter = ArrayAdapter<String>(this, android.R.layout.simple_list_item_1, items)
+    listView.adapter = adapter
```

Comment:
Can this be inlined?
```suggestion
    listView.adapter = ArrayAdapter<String>(this, android.R.layout.simple_list_item_1, items)
```